docs/source/index.rst

```diff
@@ -3,16 +3,18 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to the GameBench API documentation!
-===========================================
+Welcome to the GameBench API Client documentation!
+==================================================
 
 .. toctree::
    :maxdepth: 2
    :caption: Contents:
 
    overview
    api
+   examples
    how_to_extend
+   global_settings
    CONTRIBUTING
```

Comment:
Based on the closing statement, "Next up, Development Standards!", we should probably get 'CODE_OF_CONDUCT' and 'development_standards' swapped.

Reply:
Good catch, updated!